components/home/features/icons/CloudWithCheckmark.tsx

```diff
@@ -0,0 +1,17 @@
+export const CloudWithCheckmark = ({ className }: { className?: string }) => (
```

Comment:
Does it make sense to add this to our font-awesome library so we can access all icons in a universal way instead of looking separate places?  [This article](https://dev.to/astagi/add-custom-icons-to-font-awesome-4m67) seems to say it's possible

Reply:
I'm not sure how wild the icons are going to get. Worth a discussion with Will.